### DISCUSSION TITLE
How to open file at specified location from Ghostty?

### DISCUSSION THREAD

**Discussion by yujunz** [ghostty-org/ghostty#4022] [Q&A] [answered]
As a user I want to open a file and go to the specified location quickly from terminal, e.g.

```sh
alpha.kubernetes.manifests: error in call to encoding/yaml.MarshalStream: incomplete value string:
    ../../../../../../pkg/spec/alpha.cue:51:13
    ../../../../../../pkg/spec/deploy.cue:17:15
```

See https://docs.warp.dev/features/files-and-links#files-and-links-2

**Comment by rhodes-b** [comment, marked as answer]
https://github.com/ghostty-org/ghostty/discussions/2394

will allow you to configure how different matches open different things
